x-pack/legacy/plugins/apm/public/hooks/useTransactionBreakdown.ts

```diff
@@ -6,27 +6,27 @@
 
 import { useRef } from 'react';
 import { useFetcher } from './useFetcher';
-import { callApi } from '../services/rest/callApi';
-import { getUiFiltersES } from '../services/ui_filters/get_ui_filters_es';
-import { TransactionBreakdownAPIResponse } from '../../server/lib/transactions/breakdown';
 import { useUrlParams } from './useUrlParams';
+import { loadTransactionBreakdown } from '../services/rest/apm/transaction_groups';
 
 export function useTransactionBreakdown() {
   const {
     urlParams: { serviceName, start, end, transactionName },
     uiFilters
   } = useUrlParams();
 
-  const { data, error, status } = useFetcher(async () => {
+  const {
+    data = { kpis: [], timeseries: [] },
+    error,
+    status
+  } = useFetcher(async () => {
     if (serviceName && start && end) {
-      return callApi<TransactionBreakdownAPIResponse>({
-        pathname: `/api/apm/services/${serviceName}/transaction_groups/breakdown`,
-        query: {
-          start,
-          end,
-          transactionName,
-          uiFiltersES: await getUiFiltersES(uiFilters)
-        }
+      return loadTransactionBreakdown({
+        start,
+        end,
+        serviceName,
+        transactionName,
+        uiFilters
```

Comment:
I see what you mean with the extra abstraction and it doesn't even make it more DRY.